gramdictru/package.json

```diff
@@ -10,12 +10,12 @@
   "dependencies": {
     "@types/react": "^16.7.20",
     "@types/react-dom": "^16.0.11",
-    "@types/react-infinite-scroll-component": "^4.4.0",
     "@types/styled-components": "^4.1.6",
     "axios": "^0.18.0",
     "csv-string": "^3.1.5",
```

Comment:
I thought we're not using csv-string anymore?

Reply:
apparently my first uninstall command didn't take... I have pushed up the changes